src/components/auth/GoogleButton/index.js

```diff
@@ -0,0 +1,21 @@
+import Image from 'next/image';
+import styles from './googleButton.module.scss';
+
+const GoogleButton = () => {
+  return (
+    <button type="button" className={styles.googleButton}>
+      <div className={styles.imageContainer}>
+        <Image
+          src="/svg/google.svg"
+          alt="Google icon"
+          fill
+          sizes="auto"
+          className={styles.image}
+        />
+      </div>
+      <p>Entrar com o google</p>
```

Comment:
suggestion: we don't need a `p` here because it's not a paragraph or a big text, we could use `span` or no tags instead, `button` it's already semantic for this (probably the ideal would be `a` instead of button because you will probably need to redirect to a backend route to make the google login)